index.html

```diff
@@ -396,7 +394,6 @@ <h2 class="featurette-heading">
           <img src="/images/bug-icon.png" alt="macOS" /><br /><br />
           <p>
             <a
-              id="macos-download"
```

Comment:
the `id` is used to replace the `href` with the proper link.

Reply:
The changes are for the area in the image, these are static links with the same duplicate ID as the download area. I checked again, there is no effect on the links.

![image](https://user-images.githubusercontent.com/8020386/92227349-bc6f9f80-eed8-11ea-9e63-0fa63e582227.png)

Reply:
sorry, you're right!